src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2072,8 +2073,17 @@ private function registerRetryableHttpClient(array $options, string $name, Conta
             $retryStrategy = new Reference($options['retry_strategy']);
         } else {
             $retryStrategy = new ChildDefinition('http_client.abstract_retry_strategy');
+            $codes = [];
+            foreach ($options['http_codes'] as $code => $codeOptions) {
+                if (empty($codeOptions['methods'])) {
```

Comment:
we try to not use `empty()` when possible, can't we use something else?

Reply:
replaced by `!$codeOptions['methods']`